### PR TITLE
Updates from SEO report

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR fixes redirected links and broken links that were returned from SEO reports and an Integrity scan 🚀 

### Checklist

- [x] I have added a label to this PR 🏷️

### CODE REVIEW

File: tutorials/integrations/supra.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Supra uses a pull model as a customized approach that publishes price data upon
 2. Smart contracts are utilized for cryptographically verifying and writing the latest price data on-chain, where it lives on immutable ledgers, using [Supra's Pull contract](https://supra.com/docs/data-feeds/pull-model){target=\_blank}
```
Comment (themacexpert) [inline tutorials/integrations/supra.md:23]:
```suggestion
2. Smart contracts are utilized for cryptographically verifying and writing the latest price data on-chain, where it lives on immutable ledgers, using [Supra's Pull contract](https://docs.supra.com/oracles/data-feeds/pull-oracle){target=\_blank}
```

**Review by themacexpert — CHANGES_REQUESTED** [review]
Just one quick thing

**Comment by eshaben** [comment]
Good catch! Thanks!

**Review by themacexpert — APPROVED** [review]
LGTM!

**Pull request merged**